source/Willow/JSObject.extension.st

```diff
@@ -1,5 +1,11 @@
 Extension { #name : #JSObject }
 
+{ #category : #'*Willow' }
+JSObject >> constants [
+
```

Comment:
This sort of shortcut might open the possibilities for the confusing situations mentioned in other comments.